docs/machine-learning/tutorials/taxi-fare.md

```diff
@@ -157,69 +163,68 @@ pipeline.Add(new ColumnConcatenator("Features",
                                     "PaymentType"));
 ```
 
-Notice that the `trip_time_in_secs` column isn't included. You already determined that it isn't a useful prediction feature.
+Notice that the `TripTime` column, which corresponds to the `trip_time_in_secs` column in the data set, isn't included. You already determined that it isn't a useful prediction feature.
 
 > [!NOTE]
-> These steps must be added to Pipeline in the order specified above for successful execution.
+> These steps must be added to the pipeline in the order specified above for successful execution.
 
 ## Choose a learning algorithm
 
-After adding the data to the pipeline and transforming it into the correct input format, you select a learning algorithm (**learner**). The learning algorithm trains the model. You chose a **regression task** for this problem, so you add a learner called `FastTreeRegressor()` to the pipeline that utilizes **gradient boosting**.
+After adding the data to the pipeline and transforming it into the correct input format, you select a learning algorithm (**learner**). The learning algorithm trains the model. You chose a **regression task** for this problem, so you add a <xref:Microsoft.ML.Trainers.FastTreeRegressor> learner that utilizes **gradient boosting** to the pipeline.
```

Comment:
Perhaps it's worth mentioning that there are other learners that can also be used for regression? This makes it sound like `FastTreeRegressor` is the only option.

Reply:
That's a good suggestion. However I suggest to add it later, when the topic that describes learners is ready. Then, there is something to reference. What do you think?

Reply:
That works as well. My only concern is that if someone does not read all the way to the end, they might leave with the impression that it is the only learner. Perhaps just change the wording a bit and expand on it later?